pkg/controller/operandrequest/reconcile_operator.go

```diff
@@ -161,8 +159,38 @@ func (r *ReconcileOperandRequest) updateSubscription(cr *operatorv1alpha1.Operan
 	return nil
 }
 
+func preDeleteCheck(operandName string, reconcileReq reconcile.Request, registryInstance *operatorv1alpha1.OperandRegistry) (int, int) {
+	klog.V(4).Infof("Pre-check for delete subscription: $s", operandName)
+	operatorStatus := registryInstance.Status.OperatorsStatus[operandName]
+	if operatorStatus.Phase != operatorv1alpha1.OperatorReady {
+		pos := registryInstance.GetReconcileRequest(operandName, reconcileReq)
```

Comment:
how about s/pos/rrs

rr means reconcile request

Reply:
looks good, I update the code to use the short name